projects/sdk/src/lib/tokens.ts

```diff
@@ -46,120 +46,168 @@ export class Tokens {
     this.map = new Map();
 
     ////////// Ethereum //////////
+    const cid = this.sdk.chainId;
```

Comment:
We have plenty of storage :trollface: we can afford whole words (again, not blocking)

As in, `chainId` and `providerOrSigner`. We shouldn't have to stop and think about abbreviations when reading code.